<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminManageMetroAddStation</class>
 <widget class="QDialog" name="AdminManageMetroAddStation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>700</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #BDBDBD;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="holder_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>7</y>
      <width>181</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #D9D9D9;
border-radius: 12;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Metro_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>640</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>660</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>BAXAUTRIAL</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Egypt Metro</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>645</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>BAXAUTRIAL</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #FFC200;
border-radius: 12;
color: black;</string>
    </property>
    <property name="text">
     <string>Log out</string>
    </property>
   </widget>
   <widget class="QLabel" name="username_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>180</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>BAXAUTRIAL</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Admin</string>
    </property>
   </widget>
   <widget class="QLabel" name="userpic_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>141</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #D9D9D9;
border-radius: 12;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>250</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>BAXAUTRIAL</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #BDBDBD;
border-radius: 1;</string>
    </property>
    <property name="text">
     <string>Home Page</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>330</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>BAXAUTRIAL</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #D9D9D9;</string>
    </property>
    <property name="text">
     <string>Manage Metro</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>470</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>BAXAUTRIAL</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Subscription</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_21">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>310</y>
      <width>451</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #D9D9D9;
border-radius: 1;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_22">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>380</y>
      <width>451</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #BDBDBD;
border-radius: 1;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_23">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>450</y>
      <width>451</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #BDBDBD;
border-radius: 1;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>451</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #BDBDBD;
border-radius: 1;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_25">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>520</y>
      <width>451</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #BDBDBD;
border-radius: 1;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_29">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>540</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>BAXAUTRIAL</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Stations</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_30">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>400</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>BAXAUTRIAL</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tickets</string>
    </property>
   </widget>
   <widget class="QLabel" name="homepic_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>60</width>
      <height>55</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #D9D9D9;
border-radius: 12;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ManageMetro">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>60</width>
      <height>55</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #BDBDBD;
border-radius: 12;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Subscribtion">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>60</width>
      <height>55</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #D9D9D9;
border-radius: 12;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Station">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>60</width>
      <height>55</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #D9D9D9;
border-radius: 12;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Ticket">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>60</width>
      <height>55</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #D9D9D9;
border-radius: 12;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>449</x>
      <y>-51</y>
      <width>1200</width>
      <height>751</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QLabel" name="label_43">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>70</y>
       <width>321</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>36</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Adding Station</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_44">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>160</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>24</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Station Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_45">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>240</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>24</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Line:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_50">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>400</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>24</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Line:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_51">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>320</y>
       <width>261</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>24</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Transition:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_57">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>480</y>
       <width>291</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>24</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Station Before:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_58">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>560</y>
       <width>281</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>24</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Station After:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="name_16">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>560</y>
       <width>421</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
border-radius: 12;
color: black;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="name_17">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>400</y>
       <width>421</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
border-radius: 12;
color: black;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="name_18">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>480</y>
       <width>421</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
border-radius: 12;
color: black;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="name_19">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>160</y>
       <width>421</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
border-radius: 12;
color: black;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="name_20">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>320</y>
       <width>421</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
border-radius: 12;
color: black;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="name_21">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>240</y>
       <width>421</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
border-radius: 12;
color: black;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>670</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>BAXAUTRIAL</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #FFC200;
border-radius: 12;
color: black;</string>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_15">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #BDBDBD;
border-radius: 1;
border-image: url(:/images/img/left-arrow.png) 0 0 0 0 stretch stretch;
width: 50;
length: 50;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <zorder>pushButton_21</zorder>
   <zorder>ManageMetro</zorder>
   <zorder>pushButton_10</zorder>
   <zorder>homepic_2</zorder>
   <zorder>pushButton_23</zorder>
   <zorder>pushButton_22</zorder>
   <zorder>holder_2</zorder>
   <zorder>Metro_2</zorder>
   <zorder>label_20</zorder>
   <zorder>pushButton_11</zorder>
   <zorder>username_2</zorder>
   <zorder>userpic_2</zorder>
   <zorder>label_19</zorder>
   <zorder>label_25</zorder>
   <zorder>label_27</zorder>
   <zorder>pushButton_25</zorder>
   <zorder>label_29</zorder>
   <zorder>label_30</zorder>
   <zorder>Subscribtion</zorder>
   <zorder>Station</zorder>
   <zorder>Ticket</zorder>
   <zorder>groupBox_2</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
